ReactiveCocoa/Swift/ObjectiveCBridging.swift

```diff
@@ -184,13 +184,13 @@ extension SignalType where Value: OptionalType, Value.Wrapped: AnyObject, Error:
 		return RACSignal.createSignal { subscriber in
 			let selfDisposable = self.observe { event in
 				switch event {
-				case let .Next(value):
-					subscriber.sendNext(value.optional)
-				case let .Failed(error):
-					subscriber.sendError(error)
-				case .Completed:
-					subscriber.sendCompleted()
-				case .Interrupted:
+				case let .next(value):
+					subscriber!.sendNext(value.optional)
```

Comment:
Could be a consequence of https://github.com/apple/swift-evolution/blob/master/proposals/0054-abolish-iuo.md.